src/services/productsAPI.js

```diff
@@ -0,0 +1,3 @@
+import axios from 'axios';
+
+export const getAllProducts = () => (axios.get('http://demo4774019.mockable.io/reactTask/getProducts'))
```

Comment:
Actually, it's better to use native JS tools, alternative for `axios` - `fetch`. Your bundle will be a little bit tiny. But it's cool that you move this method in special service ;)